.github/workflows/rust.yml

```diff
@@ -296,7 +296,7 @@ jobs:
   test-datafusion-pyarrow:
     name: cargo test pyarrow (amd64)
     needs: [linux-build-lib]
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Pinning the image is only a temporary solution.
Obviously GitHub is not going to host those pinned images for ever.
Is there a follow up ticket to fix this?

Reply:
Filed https://github.com/apache/datafusion/issues/14776